src/masoniteorm/schema/Blueprint.py

```diff
@@ -398,8 +398,16 @@ def enum(self, column, options=None, nullable=False):
         for option in options:
             new_options += "'{}',".format(option)
         new_options = new_options.rstrip(",")
+
         self._last_column = self.table.add_column(
-            column, "enum", length=new_options, nullable=nullable
+            column, "enum", length=255, nullable=nullable
+        )
+
+        self.table.add_constraint(
```

Comment:
My only issue with this PR is adding a dedicated constraint type here. Won't things like sqlite, mssql and mysql need to specially handle this? What happens when those databases don't have check constraints?

It's past midnight right now but I'm actually confused on why some of those tests didn't fail lol ...

Reply:
I thought we were just going to add a dedicated column constraint. I did something like this in #363

Reply:
@josephmancuso by that I saw, all databases ( that we support ) had the `check` constraint handled equally. This change  define in `table level` and don't `column level`. 

But I can change to what you do in SQLite.